### PR TITLE
Add 'Chefkoch GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22chefkoch-gmbh%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22name%22%3A%22Chefkoch%20GmbH%22%2C%22address%22%3A%22Rheinwerk%203%5CnJoseph-Schumpeter-Allee%2033%5Cn53227%20Bonn%22%2C%22phone%22%3A%220228%2029977%20400%22%2C%22fax%22%3A%220228%2029977%20499%22%2C%22email%22%3A%22datenschutz%40chefkoch.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.chefkoch.de%2Fmagazin%2Fdatenschutz.html%22%2C%22https%3A%2F%2Fwww.chefkoch.de%2Fimpressum.php%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1344%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/chefkoch-gmbh.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "chefkoch-gmbh",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Chefkoch GmbH",
+    "address": "Rheinwerk 3\nJoseph-Schumpeter-Allee 33\n53227 Bonn",
```
Comment (moriarty-jim) [inline companies/chefkoch-gmbh.json:7]:
```suggestion
    "address": "Rheinwerk 3\nJoseph-Schumpeter-Allee 33\n53227 Bonn\nDeutschland",
```
added country

File: companies/chefkoch-gmbh.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "phone": "0228 29977 400",
```
Comment (moriarty-jim) [inline companies/chefkoch-gmbh.json:8]:
```suggestion
    "phone": "+49 228 29977400",
```
reformat phone

File: companies/chefkoch-gmbh.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "fax": "0228 29977 499",
```
Comment (moriarty-jim) [inline companies/chefkoch-gmbh.json:9]:
```suggestion
    "fax": "+49 228 29977499",
```
reformat fax

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
some minor fixes

**Review by baltpeter — APPROVED** [review]
Thanks everyone!

**Pull request merged**